azure-local/manage/sdn-troubleshooting.md

```diff
@@ -90,4 +92,12 @@ The moc-operator network security group service returned an error while reconcil
   
 ## Next steps
 
-- Review FAQ article about Azure Local SDN enabled by Azure Arc.
```

Comment:
I noticed that we are missing the link to the FAQ. Can we add that please? Thanks!

Reply:
Added link to the FAQs.